open-metadata-implementation/user-interfaces/presentation-server/nodejs/src/client/components/TypeExplorer/README.md

```diff
@@ -0,0 +1,76 @@
+## Type Explorer User Interface
+
+The Type Explorer (Tex) interface provides a way to explore the Egeria type system. It can be used to retrieve the types supported by a chosen server. The type information is then displayed both textually (on the left side) and diagrammatically (on the right side) with a choice of diagrams. One of the diagrams shows the entity inheritance tree and the other diagram shows the types of relationships that an entity type supports.
+
+![Tex Interface](image1)
+
+### Deployment
+The Tex interface runs under the Presentation Server component of Egeria, and it uses the Type Explorer (Tex) View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Server please refer to their readme information.
```

Comment:
Presentation Server and View Server please refer to their readme information. I suggest adding links

Reply:
Good point thanks - I had meant to do this. Should be fixed in next update to PR.